src/main/java/org/broadinstitute/consent/http/service/DaaService.java

```diff
@@ -102,6 +110,27 @@ public DataAccessAgreement findById(Integer daaId) {
     throw new NotFoundException("Could not find DAA with the provided ID: " + daaId);
   }
 
+  public void sendDaaRequestEmails(User user, Integer daaId) throws Exception {
+    try {
+      Institution institution = institutionDAO.findInstitutionWithSOById(user.getInstitutionId());
```

Comment:
Just thought of this. There is a corner case of a user not having an institution yet. We should error out in the case that the user doesn't have an institution. This can happen if a user signs in, goes to their profile page, and never chooses an institution. I don't anticipate the user ever seeing UI that would trigger this code path, but we should be protective both here and in the resource class as well.